contracts/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @chainlink/contracts CHANGELOG.md
 
+## Unreleased
+
+### Changed:
+- Updated `Denominations` in `./src/<version>` to include additional fiat currencies.
```

Comment:
@yosriady Could you put this under the title below, please?

0.2.2 - Unreleased

Reply:
@alexroan Updated!